common/tests/call_once_int/CMakeLists.txt

```diff
@@ -16,5 +16,5 @@ set(${theseTestsName}_cpp_files
 set(${theseTestsName}_h_files
 )
 
-build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal)
+build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal pal_ll_interfaces c_pal_ll)
```

Comment:
>pal_ll_interfaces c_pal_ll [](http://example.com/codeflow?start=96&length=26)

Does it work without this? Since c_pal should already bring in the c_pal_ll lib. I'd remove these 2 if it still builds otherwise ok to keep.

Reply:
it build fine without this, removed now